main.go

```diff
@@ -388,6 +389,32 @@ func main() {
 		log.Donef("Using system installed CocoaPods version")
 	}
 
+	// Check ruby version
+	// Run this logic only in CI environment when the ruby was installed via rbenv for the virtual machine
+	if os.Getenv("CI") == "true" && rubycommand.RubyInstallType() == rubycommand.RbenvRuby {
+		fmt.Println()
+		log.Infof("Check selected ruby is installed")
+
+		rubyInstalled, rversion, err := rubycommand.IsSpecifiedRbenvRubyInstalled(configs.SourceRootPath)
+		if err != nil {
+			log.Errorf("Failed to check if selected ruby is installed, error: %s", err)
+		}
+
+		if !rubyInstalled {
+			log.Errorf("Ruby %s is not installed", rversion)
+			fmt.Println()
+
+			cmd := command.New("rbenv", "install", rversion).SetStdout(os.Stdout).SetStderr(os.Stderr)
+			log.Donef("$ %s", cmd.PrintableCommandArgs())
+			if err := cmd.Run(); err != nil {
+				log.Errorf("Failed to install ruby version %s, error: %s", rversion, err)
```

Comment:
Ruby is a name, it should start with capital letter.

Reply:
Fixed